### ISSUE TITLE
Exactly how do I make the jar? The instructions are too vague

### ISSUE THREAD

**Issue by Anonymous3-a** [SimonSchubert/LinuxCommandLibrary#13]

**Comment by SimonSchubert** [comment]
You are right I will improve the instructions. Easiest way is to open the project in IntelliJ and execute `:cli:buildFatJar2` gradle task

**Comment by Anonymous3-a** [comment]
Nice! (Also, could you provide provide precompiled binaries?)

**Comment by SimonSchubert** [comment]
I just stumbled over this old issue. .jar are automatically attached to all new git releases by now
